NitroxPatcher/Patches/Dynamic/uGUI_SceneIntro_HandleInput_Patch.cs

```diff
@@ -1,22 +1,72 @@
-﻿using System.Reflection;
+﻿using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
 using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.PlayerLogic;
 using NitroxModel.DataStructures.GameLogic;
 using NitroxModel.Helper;
+using UnityEngine;
 
 namespace NitroxPatcher.Patches.Dynamic;
 
 public sealed partial class uGUI_SceneIntro_HandleInput_Patch : NitroxPatch, IDynamicPatch
 {
-    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((uGUI_SceneIntro t) => ((IInputHandler) t).HandleInput());
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((uGUI_SceneIntro t) => ((IInputHandler)t).HandleInput());
 
-    public static void Postfix(uGUI_SceneIntro __instance, ref bool __result)
+    public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
     {
-        if (!__result)
+        return new CodeMatcher(instructions)
+               // Remove uGUI_SceneIntro.Stop() and use our HandleButtonHeld feedback as return value
```

Comment:
I like that we add this comment to say what it is actually trying to achieve. Do we want to follow the style we have seen in others with a bit more of a format? Just thinking then it is the same across other prs. Happy to not as I think this is clear but might be more beneficial for others

Something like 
Line before
Line - removed
New line -inserted
Existing line after